src/js/tabs/osd.js

```diff
@@ -1114,13 +1122,23 @@ OSD.constants = {
 OSD.searchLimitsElement = function (arrayElements) {
     // Search minimum and maximum
     var limits = { minX: 0, maxX: 0, minY: 0, maxY: 0 };
-
-    arrayElements.forEach(function (valor, indice, array) {
-        limits.minX = Math.min(valor.x, limits.minX);
-        limits.maxX = Math.max(valor.x, limits.maxX);
-        limits.minY = Math.min(valor.y, limits.minY);
-        limits.maxY = Math.max(valor.y, limits.maxY);
-    });
+    if ((arrayElements.length > 0) && (arrayElements[0].constructor === String)) {
+	limits.maxY = arrayElements.length;
```

Comment:
I think you should check the indentation settings in the editor that you are using. Betaflight coding style states 4 spaces for indentation, and there are still plenty of tabs in your changes: https://github.com/betaflight/betaflight/blob/master/docs/development/CodingStyle.md#indentation